src/mod_antispam_dump.erl

```diff
@@ -126,20 +125,20 @@ open_dump_file(Name) ->
             undefined
     end.
 
--spec close_dump_file(undefined | file:io_device(), filename()) -> ok.
-close_dump_file(undefined, false) ->
+-spec close_dump_file(undefined | file:io_device()) -> ok.
+close_dump_file(undefined) ->
     ok;
-close_dump_file(Fd, Name) ->
+close_dump_file(Fd) ->
     case file:close(Fd) of
         ok ->
-            ?DEBUG("Closed ~s", [Name]);
+            ?DEBUG("Closed ~p", [Fd]);
         {error, Reason} ->
-            ?ERROR_MSG("Cannot close ~s: ~s", [Name, file:format_error(Reason)])
+            ?ERROR_MSG("Cannot close ~p: ~s", [Fd, file:format_error(Reason)])
     end.
 
--spec reopen_dump_file(file:io_device(), binary(), binary()) -> file:io_device().
-reopen_dump_file(Fd, OldDumpFile, NewDumpFile) ->
-    close_dump_file(Fd, OldDumpFile),
+-spec reopen_dump_file2(file:io_device(), binary()) -> file:io_device().
```

Comment:
That‘s bad naming, please make it more telling what the difference is between those two functions. Was that change even necessary? 

Anyway, you could name this one `reopen_dump_file_by_fd` or name this one `reopen_dump_file` and name what‘s `reopen_dump_file` now to `reopen_dump_file_for_host` or similiar. I think, I like the latter more.

Reply:
>  Was that change even necessary?

When stopping the module, it calls to close the dump file, for that it attempts to get the module options to know the dump path... but the module is no longer available to get the module options!

The solution I found is to not consult module options once it's getting stopped.

But now that we have prep_stop, instead of that solution, it's possible to close the dump file during prep_stop, knowing the module options as previously...